packages/wallet-connectors/src/WalletConnect.ts

```diff
@@ -327,6 +471,20 @@ export class WalletConnectConnection implements WalletConnection {
     }
 }
 
+function getChainId({ name }: Network): string {
+    return `${WALLET_CONNECT_SESSION_NAMESPACE}:${name}`;
+}
+
+/**
+ * Describes the scope of a connection to a wallet through wallet connect
+ */
+export type WalletConnectConnectionScope = {
+    /** Which methods to request permission for */
+    methods: WalletConnectMethod[];
+    /** Which events to request permission for */
+    events: WalletConnectEvent[];
+};
```

Comment:
I can see how "scope" makes sense, but still think we should go with standard WC terminology, i.e. "namespace": It's not great, but it's what you would look for and would understand the easiest when trying to dig into some problem.